mender/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Mender Helm chart
 
+## Version 5.10.0
+* Upgrade Traefik to v3.1.0
```

Comment:
```suggestion
* Upgrade Traefik to v3.1.1
```

Reply:
Bumped it to `v3.1.2` and updated the changelog.